Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -66,6 +81,23 @@ public override IEnumerable<ISetting> GetSettings()
             yield return stringTemplateSetting;
         }
 
+        private void QueryHelperLink_Click(object sender, EventArgs e)
+        {
+            if (string.IsNullOrWhiteSpace(urlSettings.CustomControl.Text))
+            {
+                MessageBox.Show(null, "Please fill the jira url to open the query helper", "Jira Url not filled", MessageBoxButtons.OK, MessageBoxIcon.Warning);
+                return;
+            }
+            try
+            {
+                Process.Start(urlSettings.CustomControl.Text + "/secure/IssueNavigator.jspa?mode=show&createNew=true");
+            }
+            catch (Exception)
+            {
+                MessageBox.Show(null, "The url is not valid", "Error opening Jira", MessageBoxButtons.OK, MessageBoxIcon.Error);
```

Comment:
Proposing to tweak the messages a little:
* "Error opening Jira" -> "Unable to open Jira query helper"
* "The url is not valid" -> `ex.Message` as we don't really know what the error may be (network is unavailable, host is down etc)

thoughts?